types/umami/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for umami 0.1
+// Project: https://github.com/mikecao/umami
+// Definitions by: Noah Overcash <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/umami/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me

Reply:
Oops, didn't even notice that.